OpenShift-Deployment-Guide.md

```diff
@@ -0,0 +1,56 @@
+# Deploying to OpenShift
+
+If you deploy your applications to Heroku you only can upload 5 applications, if you want to deploy a new one you need to verify you account with your credit card.
+
+![Heroku Error](http://i.imgur.com/fY2be3a.jpg)
+
+After some search at Google I found OpenShift. Then I went to Quincy Larson to know if was any problem to deploy to OpenShift instead of Heroku, he said it was Ok and ask me if I could document the process, and here it is...
```

Comment:
> Then I went to Quincy Larson to know if was any problem to deploy to OpenShift instead of Heroku, he said it was Ok and ask me if I could document the process, and here it is...

This is a Wiki article. No need to mention details like that.